web/template/admin/admin_tabs/workers.gohtml

```diff
@@ -1,7 +1,6 @@
 {{define "workers"}}
-    <h2 class="text-2xl text-1 my-auto mt-4 font-medium">Worker Overview</h2>
-    <div class="dark:border-secondary bg-gray-100 rounded-md overflow-x-auto mb-6 my-3 shadow-md dark:bg-secondary-lighter">
-        <table class="table-auto w-full rounded-sm">
+    <div class="form-container">
+        <table class="table-auto w-full form-container-title">
```

Comment:
I think this broke the table a little bit (See Screenshot)

<img width="1104" alt="Screenshot 2023-04-03 at 13 02 25" src="https://user-images.githubusercontent.com/29633518/229491574-ca059ef9-1705-4d12-95ce-eb1e4a369e6d.png">

Reply:
Good catch, probably overlooked something during merging.